matchbox_demo/src/main.rs

```diff
@@ -145,25 +142,26 @@ fn lobby_cleanup(query: Query<Entity, With<LobbyUI>>, mut commands: Commands) {
 fn lobby_system(
     mut app_state: ResMut<State<AppState>>,
     args: Res<Args>,
-    mut socket: ResMut<Option<WebRtcSocket>>,
+    mut socket: ResMut<SocketResource>,
     mut commands: Commands,
     mut query: Query<&mut Text, With<LobbyText>>,
 ) {
-    let socket = socket.as_mut();
+    if socket.0.is_none() {
+        return;
+    }
```

Comment:
**note** this can probably be removed given we set the resource when entering the state

Reply:
Yep, go ahead 👍